### PR TITLE
refine runExtendedSQL

### PR DESCRIPTION
**Pull request by sperlingxx**
This PR is about refining codegen preparation/codegen part of `runExtendedSQL`, make it more reusable (for `analyze` and more possible stages in future).

### CODE REVIEW

File: sql/executor.go [hunk 1]
```diff
@@ -466,6 +481,20 @@ func pred(wr *PipeWriter, pr *extendedSelect, db *DB, cwd string, modelDir strin
 }
 
 func analyze(wr *PipeWriter, es *extendedSelect, db *DB, cwd string, modelDir string) error {
+	//pr, fts, e := loadModelMeta(es, db, cwd, modelDir)
```
Comment (Yancey0623) [inline sql/executor.go:484]: Remove the commented code?

**Review by Yancey0623 — COMMENTED** [review]
Please clean up the code and LGTM.

File: sql/executor.go (hunk 1 above)
Comment (weiguoz) [reply sql/executor.go:484]: It is a sample code for how to use it.
Next PR will remove it.

**Review by weiguoz — APPROVED** [review]
LGTM

**Pull request merged**